src/focus.service.ts

```diff
@@ -521,11 +521,10 @@ export class FocusService {
       return false;
     }
 
-    if (el.tabIndex > -1) {
-      return true;
+    if (el.tabIndex < 0) {
+      return false;
     }
```

Comment:
Are you sure that every div has a tab index? That's definitely far away from what the web standards say should be happening.